File: csharp/coin/dnmcprgrmng/fbnc/brtfrc/SolutionTest.cs
using NUnit.Framework;

namespace dnmcprgrmng.fbnc.brtfrc
{
    public class Tests
    {
        Solution solution;

        [SetUp]
        public void Setup()
        {
            solution = new Solution();
        }

        [Test]
        public void Correctness()
        {
            Assert.That(solution.fibo(1), Is.EqualTo(1));
            Assert.That(solution.fibo(2), Is.EqualTo(1));
            Assert.That(solution.fibo(3), Is.EqualTo(2));
            Assert.That(solution.fibo(4), Is.EqualTo(3));
            Assert.That(solution.fibo(5), Is.EqualTo(5));
            Assert.That(solution.fibo(6), Is.EqualTo(8));
            Assert.That(solution.fibo(7), Is.EqualTo(13));
            Assert.That(solution.fibo(8), Is.EqualTo(21));
            Assert.That(solution.fibo(9), Is.EqualTo(34));
        }
    }
}

File: java/dnmcprgrmng/test/fbnc/brtfrc/SolutionTest.java
package fbnc.brtfrc;

import org.junit.*;

import static org.hamcrest.MatcherAssert.*;
import static org.hamcrest.Matchers.*;

public class SolutionTest {
  Solution solution;

  @Before
  public void BeforeEach() {
    solution = new Solution();
  }

  @Test
  public void correctness() {
    assertThat(solution.fibo(0), is(equalTo(0)));
    assertThat(solution.fibo(1), is(equalTo(1)));
    assertThat(solution.fibo(2), is(equalTo(1)));
    assertThat(solution.fibo(3), is(equalTo(2)));
    assertThat(solution.fibo(4), is(equalTo(3)));
    assertThat(solution.fibo(5), is(equalTo(5)));
    assertThat(solution.fibo(6), is(equalTo(8)));
    assertThat(solution.fibo(7), is(equalTo(13)));
    assertThat(solution.fibo(8), is(equalTo(21)));
    assertThat(solution.fibo(9), is(equalTo(34)));
  }
}
